### PR TITLE
Use flexbox

### PR DESCRIPTION
**Pull request by stantoncbradley**
#review @raylendup

### CODE REVIEW

File: ProgressBar.js [hunk 1]
```diff
@@ -3,64 +3,70 @@ import {
...
     });
 
     return (
-      <View style={[styles.background, this.props.backgroundStyle, this.props.style]}>
-        <Animated.View style={[styles.fill, this.props.fillStyle, { width: fillWidth }]}/>
+      <View
+        style={[styles.background, this.props.backgroundStyle, this.props.style]}
+        onLayout={(event) => {
+          if (!this.props.style.width) {
+            this.setState({ calculatedWidth: event.nativeEvent.layout.width });
```
Comment (raylendup) [inline ProgressBar.js:56]: nice!

**Review by raylendup — APPROVED** [review]
Looks good.

**Comment by raylendup** [comment]
#release

File: ProgressBar.js (hunk 1 above)
Comment (stantoncbradley) [reply ProgressBar.js:56]: hehe pulled that from an open pr.  gonna be super useful tho!

**Pull request merged**